parastell/source_mesh.py

```diff
@@ -282,18 +281,20 @@ def create_vertices(self):
 
                     vert_idx += 1
 
-        self.verts = self.mbc.create_vertices(self.coords)
+        self.add_vertices(self.coords)
 
-    def _source_strength(self, tet_ids):
+    def _compute_tet_data(self, tet_ids, tet):
         """Computes neutron source strength for a tetrahedron using five-node
```

Comment:
Add something to the docstring to clarify that this method sets the value of the tag, as well...